File: 13458_시험감독.js
const [[N],A,[B,C]] = `5
10 9 10 9 10
7 2`.toString().trim().split('\n').map(el => el.split(' ').map(Number));

    const solution = () => {

        let answer = 0;

        for(let i=0; i<N; i++){
            const students = A[i];

            //총감독관 1명은 무조건 필요
            answer += 1;

            const restStudent = students - B;

            if (restStudent > 0){
                answer += Math.ceil(restStudent / C);
            }
        }

        return answer;
    }

    console.log(solution())

File: _python/13458_시험감독.py
import math

N = int(input())
A = list(map(int, input().split()));
B,C = map(int, input().split());

def solution() :
    
    answer = 0

    for i in range(N) :
        students = A[i]

        answer += 1

        restStudent = students - B;

        if restStudent > 0 :
            answer += math.ceil(restStudent / C);
    
    return answer

print(solution());